pkg/apis/serving/k8s_validation_test.go

```diff
@@ -1080,7 +1079,16 @@ func TestContainerValidation(t *testing.T) {
 				ReadOnly:  true,
 			}},
 		},
-		volumes: sets.NewString("the-name"),
+		volumes: map[string]corev1.Volume{
+			"the-name": {
+				Name: "the-name",
+				VolumeSource: corev1.VolumeSource{
+					ConfigMap: &corev1.ConfigMapVolumeSource{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: "test-cm",
+						}},
+				},
+			}},
```

Comment:
total nit, but I _think_ for consistency the formatting here should be:

```suggestion
		volumes: map[string]corev1.Volume{
			"the-name": {
				Name: "the-name",
				VolumeSource: corev1.VolumeSource{
					ConfigMap: &corev1.ConfigMapVolumeSource{
						LocalObjectReference: corev1.LocalObjectReference{
							Name: "test-cm",
						},
					},
				},
			},
		},
```

(basically line the braces up under the first column of where the opening brace is)

Reply:
ok